Source/Xamarin/Prism.Forms.Tests/Services/PageDialogServiceFixture.cs

```diff
@@ -20,68 +20,273 @@ public PageDialogServiceFixture()
         [Fact]
         public async Task DisplayActionSheetNoButtons_ShouldThrowException()
         {
-            var service = new PageDialogServiceMock("cancel", _applicationProvider);
-            var argumentException = await Assert.ThrowsAsync<ArgumentException>(() => service.DisplayActionSheetAsync(null, null));
-            Assert.Equal(typeof(ArgumentException), argumentException.GetType());
+            var service = new PageDialogServiceMock( "cancel", _applicationProvider );
+            var argumentException = await Assert.ThrowsAsync<ArgumentException>( () => service.DisplayActionSheetAsync( null, null ) );
+            Assert.Equal( typeof( ArgumentException ), argumentException.GetType() );
+        }
+
+        #region Tests using Action
+
+        private async Task DisplayActionSheet_PressButton_UsingAction( string text )
```

Comment:
My normal spacing is `( someVar )`, I generally try to reverse that for Prism where the standard is `(someVar)`. The new EditorConfig settings in VS2017 will probably help a lot with that sort of thing in the future. Thanks for the catch, I'll update that.